lib/SmartyPlugins/LMSSmartyPlugins.php

```diff
@@ -163,6 +163,7 @@ public static function divisionSelectionFunction(array $params, $template)
         $label = isset($params['label']) ? $params['label'] : null;
         $name = isset($params['name']) ? $params['name'] : 'division';
         $id = isset($params['id']) ? $params['id'] : $name;
+        $icon = empty($params['icon']) ? null : $params['icon'];
```

Comment:
to samo - empty() na niestniejących elemncie tablicy nie powoduje warninga od PHP 8.0.
Przejrzyj wszystko pod tym kątem - dużo pracy włożyłem ostatnio w walkę z takimi wargninami i szkoda, żeby nowe dochodziły.

Reply:
Sporo wcześniej niż PHP8.0 nie pluje błędami gdy zmienna nie istnieje, dlatego tak chętnie stosuję tą próbę - dziękuję za cenne wskazówki. Tutaj brak warningów